modules/serverless-installing-cli-linux-ibm-z_rpm.adoc

```diff
@@ -0,0 +1,53 @@
+// Module is included in the following assemblies:
+//
+// serverless/knative-client.adoc
+
+[id="installing-cli-linux_ibm_z_rpm_{context}"]
+
+== RPM Installation
+
+For Red Hat Enterprise Linux (RHEL), you can install `kn` as an RPM if you have an active {product-title} subscription on your Red Hat account.
+
+
+.Procedure
+. Enter the command:
++
+
+[source,terminal]
+----
+# subscription-manager register
+----
+
+. Enter the command:
++
+
+[source,terminal]
+----
+# subscription-manager refresh
+----
+
+. Enter the command:
++
+
+[source,terminal]
+----
+# subscription-manager attach --pool=<pool_id> <1>
+----
++
+<1> Pool ID for an active {product-title} subscription
+
+. Enter the command:
++
+
+[source,terminal]
+----
+# subscription-manager repos --enable="openshift-serverless-1-for-rhel-8-s390x-rpms"
+----
+
+. Enter the command:
++
+
+[source,terminal]
+----
+# yum install openshift-serverless-clients
+----
```

Comment:
Instead of saying "enter the command" for all of these, explain what the command does, e.g. "Install x thing:"